.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -14,22 +14,37 @@ body:
       description: |
         Provide a detailed description of the issue.  Include relevant information, such as:
         - The feature or configuration option you encounter the issue with.
-        - The expected behavior.
-        - The actual behavior (and how it deviates from the expected behavior, if it is not immediately obvious).
-        - Relevant Ghostty logs or other stacktraces.
-        - Relevant screenshots, screen recordings, or other supporting media (as needed).
+        - Screenshots, screen recordings, or other supporting media (as needed).
         - If this is a regression of an existing issue that was closed or resolved, please include the previous item reference (Discussion, Issue, PR, commit) in your description.
 
         >[!TIP]
         > **Not sure what information to include?**
         > Here are some recommendations:
-        > - **Input issues:** include your keyboard layout, a screenshot of the terminal inspector's logged keystrokes (Linux: <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>i</kbd>; MacOS: <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>i</kbd>), input method, Linux input method engine (IBus, Fcitx 5, or none) and its version.
+        > - **Input issues:** include your keyboard layout, a screenshot of logged keystrokes from the terminal inspector's "Keyboard" tab (Linux: <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>i</kbd>; MacOS: <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>i</kbd>), input method, Linux input method engine (IBus, Fcitx 5, or none) and its version.
```

Though an alternative is to change every quote to use smart quotes, I have no opinion on this though.
```suggestion
        > - **Input issues:** include your keyboard layout, a screenshot of logged keystrokes from the terminal inspector's “Keyboard” tab (Linux: <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>i</kbd>; MacOS: <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>i</kbd>), input method, Linux input method engine (IBus, Fcitx 5, or none) and its version.
```

---

I think we should stick with ASCII quotes.  Anecdotal, perhaps, but I find them more compatible with various platforms than smart quotes.